src/models.ts

```diff
@@ -11,6 +11,23 @@ export interface Serp {
   hotels?: Hotels;
   adwords?: { adwordsTop?: Ad[]; adwordsBottom?: Ad[] };
   availableOn?: AvailableOn[];
+  shop?: ShopCard[];
```

Comment:
Naming is wrong, use `shopResults` or `shoppingResults` and for the interface `ShopResult` or `ShoppingResult`